Amplify/Categories/Auth/Request/AuthWebUISignInRequest.swift

```diff
@@ -5,30 +5,36 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#if canImport(AuthenticationServices)
```

Comment:
That's a good point, originally we (when Chris was around) decided against excluding HostedUI files to avoid excessive compiler checks as HostedUI was supported on watchOS, but I can check if we can just exclude this one since now it's gone on both new platforms